index.html

```diff
@@ -106,3 +106,7 @@ <h3>Stop Planning</h3>
     <footer>© 2021 Travel Blog, LLC.</footer>
   </body>
 </html>
+
+  <!-- linking our JS document to our html -->
+  <script src="./main.js"></script>
+</html>
```

Comment:
Seems like an extra closing </html> tag, which you already have on line 108. Should be able to move your script tag under line 107 to have it as part of the document, or have it in the head with the `defer` attribute.